steward/src/cellars/cellar_v2_2.rs

```diff
@@ -233,6 +233,20 @@ pub fn get_encoded_function(call: FunctionCall, cellar_id: String) -> Result<Vec
 
             Ok(CellarV2_2Calls::RemovePositionFromCatalogue(call).encode())
         }
+        Function::CachePriceRouter(params) => {
+            validate_cache_price_router(&cellar_id)?;
+            log_cellar_call(
+                CELLAR_NAME,
+                &CachePriceRouterCall::function_name(),
+                &cellar_id,
+            );
+            let call = CachePriceRouterCall {
+                check_total_assets: params.check_total_assets,
+                allowable_range: params.allowable_range as u16,
```

Comment:
The proto message includes an expected address, is there a reason it's unset here?

Reply:
Ah need to fix the proto. I made it match the 2.5 cellar abi by mistake.